src/components/buttons/feedback/feedbackButton.tsx

```diff
@@ -0,0 +1,13 @@
+import Image from 'next/image'
+import type { FC } from 'react'
+import feedbackIcon from '../../../../public/icons/feedback.svg'
+import './feedbackButton.scss'
+
+export const FeedbackButton: FC = () => (
```

Comment:
Shouldn't we have avoided using FC as we discussed a few months ago? It is possible that this has changed in the meantime!